content/en/docs/demo/collector-data-flow-dashboard.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Collector Data Flow Dashboard
+---
+
+Monitoring data flow through the OpenTelemetry Collector is crucial for several
+reasons. Gaining a macro-level perspective on incoming data, such as sample
+counts and cardinality, is essential for comprehending the collector's internal
+dynamics. However, when delving into the details, the interconnections can
+become complex. The Collector Data Flow Dashboard aims to demonstrate the
+capabilities of the OpenTelemetry demo application, offering a solid foundation
+for users to build upon. Collector Data Flow Dashboard provides valuable
+guidance on which metrics to monitor. Users can tailor their own dashboard
+variations by adding necessary metrics specific to their use cases, such as
+memory_delimiter processor or other o11y data flow indicators. This demo
```

Comment:
Do we use `o11y` somewhere else in the docs?
Even though it's a known abbreviation, I'd rather have it as observability here, just to keep it simple.

Reply:
Thanks for noticing such detail. It felt strange when I put it in the first place.  I agree, I removed it now.👌🏽